uaclient/cli.py

```diff
@@ -596,6 +596,16 @@ def wrapper(*args, **kwargs):
                 logging.exception(exc.msg)
             print("{}".format(exc.msg), file=sys.stderr)
             sys.exit(exc.exit_code)
+        except Exception as exc:
+            with util.disable_log_to_console():
+                logging.exception(exc)
+            print(
+                ua_status.MESSAGE_UNEXPECTED_ERROR_DURING_OP_TMPL.format(
+                    operation="command: ua {}".format(" ".join(sys.argv[1:]))
```

Comment:
I don't know that we want to echo back the full command. It could lead to a very long line (consider that a command could be as long as `ua attach --no-auto-enable myverylongtokeninhere`), and they'll already know what they just ran. (Also, handily, we now log the arguments. ^_^)  What do you think?

Reply:
+1 dropped full command output in failure message.